src/main/java/com/osiris/autoplug/client/tasks/updater/server/TaskServerUpdater.java

```diff
@@ -403,114 +300,31 @@ private void doFabricUpdatingLogic() throws WrongJsonTypeException, IOException,
-            cache_dest.createNewFile();
-            TaskDownload download = new TaskDownload("ServerDownloader", getManager(),
-                    paperDownloadsAPI.getDownloadUrl(serverSoftware, serverVersion, latest_build_id, build_name),
-                    cache_dest);
-            download.start();
-
-            while (true) {
-                Thread.sleep(500);
-                if (download.isFinished()) {
-                    if (download.isSuccess()) {
-                        setStatus("Server update downloaded. Checking hash...");
-                        if (download.compareWithSHA256(build_hash)) {
-                            File final_dest = serverExe;
-                            if (final_dest == null)
-                                final_dest = new File(GD.WORKING_DIR + "/" + serverSoftware + "-latest.jar");
-                            if (final_dest.exists()) final_dest.delete();
-                            final_dest.createNewFile();
-                            FileUtils.copyFile(cache_dest, final_dest);
-                            setStatus("Server update was installed successfully (" + build_id + " -> " + latest_build_id + ")!");
-                            updaterConfig.server_build_id.setValues("" + latest_build_id);
-                            updaterConfig.save();
-                            setSuccess(true);
-                        } else {
-                            if (updaterConfig.server_skip_hash_check.asBoolean()) {
-                                addWarning("Note that the hash check failed for this download, but installing anyways because skip-hash-check is enabled!");
-                                File final_dest = serverExe;
-                                if (final_dest == null)
-                                    final_dest = new File(GD.WORKING_DIR + "/" + serverSoftware + "-latest.jar");
-                                if (final_dest.exists()) final_dest.delete();
-                                final_dest.createNewFile();
-                                FileUtils.copyFile(cache_dest, final_dest);
-                                setStatus("Server update was installed successfully (" + build_id + " -> " + latest_build_id + ")!");
-                                updaterConfig.server_build_id.setValues("" + latest_build_id);
-                                updaterConfig.save();
-                                setSuccess(true);
-                            } else {
-                                setStatus("Downloaded server update is broken (hash check failed). Nothing changed!");
-                                setSuccess(false);
-                            }
-                        }
-
-                    } else {
-                        setStatus("Server update failed!");
-                        setSuccess(false);
-                    }
-                    break;
-                }
+            setStatus("Starting MCServer updater...");
```

Comment:
Keep the old status message. I know you want to advertise your project, you don't need to worry about that. You can add your project under the README used libraries section.

Reply:
You can do that on your own.

Reply:
I changed the message